src/Core/LeanCode.Components.Startup/IHostBuilderExtensions.cs

```diff
@@ -110,7 +111,7 @@ public static class IHostBuilderExtensions
 
                 if (context.HostingEnvironment.IsDevelopment())
                 {
-                    loggerConfiguration.WriteTo.Console();
+                    loggerConfiguration.WriteTo.Console(formatProvider: CultureInfo.InvariantCulture);
```

Comment:
Can we remove the name of the parameter? For a single parameter it does not add to the meaning.

Reply:
I don't think we can. This function takes several parameters so it tries to convert formatProvider to LogEventLevel if we don't specify which parameter it is.

Reply:
Ahhh, got it!